pkg/services/ngalert/ngalert.go

```diff
@@ -246,19 +245,19 @@ func subscribeToFolderChanges(logger log.Logger, bus bus.Bus, dbStore api.RuleSt
 	bus.AddEventListener(func(ctx context.Context, e *events.FolderTitleUpdated) error {
 		// do not block the upstream execution
 		go func(evt *events.FolderTitleUpdated) {
-			logger.Debug("Got folder title updated event. updating rules in the folder", "folder_uid", evt.UID)
+			logger.Info("Got folder title updated event. updating rules in the folder", "folderUID", evt.UID)
```

Comment:
Correct. It happens only when the user renames a folder, which does not happen quite often. 

log on line 255 is a better fit for Info because it indicates that some rules were updated. However, I think it makes sense to switch this one back to Debug because the logs below cover all outcomes.